src/components/CheckForm/AlertsPerCheck/AlertsPerCheck.constants.tsx

```diff
@@ -3,103 +3,96 @@ import { CheckAlertCategory, CheckAlertType, CheckType, ThresholdUnit } from 'ty
 export interface PredefinedAlertInterface {
   type: CheckAlertType;
   name: string;
+  description: string;
   unit: ThresholdUnit;
   category: CheckAlertCategory;
-  default?: number;
+  query: string;
+  defaultValues: {
+    threshold: number;
+    period?: (typeof ALERT_PERIODS)[number]['value'];
+    isSelected: boolean;
+  };
+  supportsPeriod: boolean;
 }
 
+export const ALERT_PERIODS = [
+  { label: '5 min', value: '5m' },
+  { label: '10 min', value: '10m' },
+  { label: '15 min', value: '15m' },
+  { label: '20 min', value: '20m' },
+  { label: '30 min', value: '30m' },
+  { label: '1 h', value: '1h' },
+] as const;
+
 export const GLOBAL_PREDEFINED_ALERTS: PredefinedAlertInterface[] = [
   {
     type: CheckAlertType.ProbeFailedExecutionsTooHigh,
     name: 'Probe Failed Executions Too High',
-    unit: '%',
-    category: CheckAlertCategory.SystemHealth,
-    default: 10,
+    description: `The available periods are shown based on the selected check frequency. Only periods equal to or greater than the check frequency will be available for selection.`,
+    supportsPeriod: true,
+    unit: 'no.',
+    category: CheckAlertCategory.FailedChecks,
+    defaultValues: {
+      threshold: 1,
+      period: '5m',
+      isSelected: false,
+    },
+    query: `
+    (
```

Comment:
Notice that this query has now changed. You can see the current query [here](https://github.com/grafana/synthetic-monitoring-api/blob/914e60a309b4503656e0b9ed0fe6f53017164003/internal/alerts/rules.go#L17).

Reply:
Thanks for the catch, I've updated it [here](https://github.com/grafana/synthetic-monitoring-app/pull/1064/commits/640446657fad542575e276f0653ec635b7c679c2)

Reply:
Should this: https://github.com/grafana/synthetic-monitoring-app/blob/640446657fad542575e276f0653ec635b7c679c2/src/components/CheckForm/AlertsPerCheck/AlertsPerCheck.constants.tsx#L48-L50
Be `$threshold` as it was [before](https://github.com/grafana/synthetic-monitoring-app/blob/8a2c7f294013999537c9c8277da3b2a3208e2511/src/components/CheckForm/AlertsPerCheck/AlertsPerCheck.constants.tsx#L52)? As I understand that references the value that the user set in the form when editing the check? Instead of referencing the current threshold value that is represented by the metric.

Reply:
Makes sense. Changed it 👍 Attaching screenshots with an example of how the alert configuration translates to the query.
![image](https://github.com/user-attachments/assets/baacb7ea-1342-4624-aecc-3b571c1ae8e7)
![image](https://github.com/user-attachments/assets/8ab926ab-dbd4-4d18-80fd-a9c7db26dbcb)